### ISSUE TITLE
How to train the network with my dataset? and resize the input size?

### ISSUE THREAD

**Issue by KRYSTALLJY** [lukemelas/EfficientNet-PyTorch#221]

**Comment by datduong** [comment]
I find this guide helpful. https://colab.research.google.com/drive/1Jw28xZ1NJq4Cja4jLe6tJ6_F5lCzElb4#scrollTo=RzgmGb73yFNj

It shows you how to use transforms.Compose to transform the images. 

To train, you have to put a loss function on the output of model.forward(input). The link above also shows you how to do a forward pass to get the prediction, so that you can apply loss function on it.
